float64[] joints
---
moveit_msgs/MoveItErrorCodes error_code